README.md

```diff
@@ -2,33 +2,60 @@
 
 Theme for using GitBook to publish an FAQ or Knowledge Base. This theme works perfectly with search plugins (as the [default one](https://github.com/GitbookIO/plugin-search) or [algolia](https://github.com/GitbookIO/plugin-algolia)).
 
-### Usage
+## Usage
 
 This theme requires GitBook version 3 or later.
 
 Add the theme to your book's configuration (`book.json`):
 
 ```js
 {
-    "plugins": ["theme-faq"]
+    "plugins": [
+        "theme-faq",
+        "-fontsettngs",
+        "-sharing"
+        ]
 }
 ```
 
-##### Add relations between articles
+**NOTE** `theme-faq` is not compatible with plugins that modify the toolbar (since there is no toolbar). Embedded search will not work as a result. 
 
-Suggestions for other articles can be shown at the bottom of an article, relations are specified in the YAML frontmatter of a page:
+This includes the default plugins `fontsettings` and `sharing` that need to be disabled explicitly (add a minus flag "-" before each plugin parameter).
 
+If you disable both, the search bar will work. Otherwise the javascript fails when loading the page.
+
+### Add relations between articles
+
+Suggestions for other articles can be shown at the bottom of an article. 
+
+Relationships are specified in the YAML frontmatter of a page.  Use the syntax of the markup language you are writing in.
+
+#### For Markdown Topics
 ```md
 ---
 related:
     - some/other/page.md
     - another_related_article.md
+    
 ---
 
 My article!
 ```
 
-##### Add logo to header
```

Comment:
There really should not be a difference for Markdown/AsciiDoc here. If yes, it's a bug in GitBook. So I'll just discard these changes